src/unplugin.ts

```diff
@@ -7,15 +7,23 @@ type ComponentMetaUnpluginOptions = { parser?: ComponentMetaParser } & ModuleOpt
 export const metaPlugin = createUnplugin<ComponentMetaUnpluginOptions>(
   ({ parser, ...options }) => {
     const instance = (parser || useComponentMetaParser(options)) as ComponentMetaParser
+    let _configResolved: any
 
     return {
       name: 'vite-plugin-nuxt-component-meta',
       enforce: 'post',
       async buildStart () {
+        // avoid parsing meta twice
+        if (_configResolved && !_configResolved.build.ssr) {
```

Comment:
```suggestion
        // avoid parsing meta twice in SSR
        if (_configResolved?.build.ssr) {
```

I guess maybe it's better to skip on SSR instead?